Serializer/Serializer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Serializer;
+
+/**
+ * @author Christian Flothmann <christian.flothmann@xabbuh.de>
+ */
+interface Serializer
+{
+    /**
+     * @param mixed  $data
+     * @param string $format
+     * @param mixed  $context
```

Comment:
this should probably be more specific than ``mixed``, abstracting the differences